app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -453,12 +464,15 @@ public static void clickAddToHome() {
     }
 
     public static void clickToolbarCapture() {
-        TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.TOOLBAR, Value.CAPTURE).queue();
+        TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.TOOLBAR, Value.CAPTURE)
+                .extra(Extra.VERSION, Integer.toString(2))
+                .queue();
     }
 
     public static void clickTopSiteOn(int index) {
         TelemetryEvent.create(Category.ACTION, Method.OPEN, Object.HOME, Value.LINK)
                 .extra(Extra.ON, Integer.toString(index))
+                .extra(Extra.VERSION, Integer.toString(2))
```

Comment:
I'm actually uncertain why we upgrade this. (We have not changed it somehow?) Maybe we want to know if the new home / browser navigation has an impact but maybe we should upgrade more? Like Home / BrowserFragment first level item clicks?